design/sd-local.md

```diff
@@ -15,6 +15,7 @@ As a result, User cannot confirm whether the result obtained on CI is the expect
 - March 4th, 2020: Added `src-url` option and updated env options
 - March 16th, 2020: Added `sudo` option
 - March 24th, 2020: Added `--local` option
```

Comment:
Is it better to delete this line because we do not use `--local` option anymore?
Or we should add some easy explanation that `--local` option has been changed to multiple cluster configs.

Reply:
I think it's better to add a message with the "--local" option deleted. because this section is the change log.

Reply:
fixed